plugins/badges-backend/src/badges.test.ts

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright 2021 Spotify AB
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import type { Entity } from '@backstage/catalog-model';
+import { InputError } from '@backstage/errors';
+import { Config, ConfigReader } from '@backstage/config';
+import { BadgeContext, BadgeFactories } from './types';
+import { createDefaultBadgeFactories } from './badges';
+
+describe('BadgeFactories', () => {
+  let badgeFactories: BadgeFactories;
+  let config: Config;
+
+  beforeAll(() => {
+    badgeFactories = createDefaultBadgeFactories();
+    config = new ConfigReader({
```

Comment:
Config can probably be created on line 25 since it's not changing between tests.

Reply:
Is that preferable for any reason?
The same goes for `badgeFactories` btw.

My reasoning was that you might get better feedback in case something blows up in the `beforeAll` setup call.. or it just follows the same pattern used elsewhere where local test data is initialized in a `beforeAll` or `beforeEach`.